package.json

```diff
@@ -27,7 +27,7 @@
     "sinon": "^14.0.0"
   },
   "peerDependencies": {
-    "serverless": "2.x"
+    "serverless": ">= 2.x <= 3.21.x"
```

Comment:
There is no reason to restrict the upper bound

Reply:
I was setting to the current version, but I can increase it. Would just `>=2.x` be good, or should it still have an upper cap of `3.x`?

Reply:
The problem with the current version is that it's not gonna be current in a few days and we'll be back to the same problem. The reason I had `2.x` is because `1.x` does not have some APIs that we use. I'd do `>= 2.x < 4.x` as we know that we are compatible with both `2.x` and `3.x` and we'll reassess when `4.x` comes out.